spec/features/delete_taxon_spec.rb

```diff
@@ -0,0 +1,142 @@
+require "rails_helper"
+
+RSpec.feature "Delete Taxon", type: :feature do
+  include ContentItemHelper
+  include PublishingApiHelper
+
+  scenario "a taxon with no children" do
+    given_a_taxon_with_no_children
+    when_i_visit_the_taxon_page
+    and_i_click_to_delete_the_taxon
+    then_i_expect_to_be_informed_that_im_about_to_delete_the_taxon
+    and_i_should_see_the_action_buttons
+    and_i_click_delete
+    then_i_expect_the_request_to_have_been_made
+  end
+
+  scenario "a parent taxon with children" do
```

Comment:
What do you think of including a couple of content items as well? That way we would be able to assert that they are also on the page together with the children.